.vscode/extensions.json

```diff
@@ -6,6 +6,7 @@
         "webhint.vscode-webhint",
         "IronGeek.vscode-env",
         "firsttris.vscode-jest-runner",
-        "ms-playwright.playwright"
+        "ms-playwright.playwright",
+        "orta.vscode-jest"

```

Comment:
This is another option for running jest tests, right? Seems like we should pick between this one and the existing "firsttris.vscode-jest-runner" on line 8?

Reply:
I find this extension to be much (MUCH) better. I didn't even notice the other one is there. I continue testing it locally, will see what i come up with.